<?xml version="1.0" ?>
<launch>

  <!--Start Learning-->
  <node pkg="mir_vision" type="learnObject" name="learnObject" output="screen" >
    <param name="object_name"               type="str"    value="Teabox" /> 
    <param name="feature"                   type="str"    value="SURF" />
    <param name="path_package"              type="str"    value="$(find mir_vision)/"/> 
    <param name="verbose"                   type="bool"   value="true" />
    <param name="gray"                      type="bool"   value="false" />
    <param name="depth"                     type="bool"   value="false" />
    <param name="keypoints"                 type="bool"   value="true" />  <!-- if false just images will be generated, otherwise keypoints are learned as well -->     
    <param name="replaceLastImage"          type="bool"   value="false" />
    <param name="binMode"                   type="bool"   value="true" />  
    <param name="maxTrainImageNumber"       type="int"    value="3" />
  </node>
</launch>
